react/Labs/ExperimentalModal/Readme.md

```diff
@@ -0,0 +1,33 @@
+### ExperimentalModal
+
+This modal brings a new UI only on Mobile for now. It changes the
+size of the Header and its margins and add border-bottom.
+It also changes, our ModalFooter to something fixed in the bottom
+and add a border-top.
+
+Not tested on "long content".
+
+```
+const ExperimentalModal = require('.').default
+
+initialState = { modalOpened: false};
+const hideModal = () => setState({ modalOpened: false });
+
+<div>
+  <button onClick={()=>setState({ modalOpened: !state.modalOpened })}>
+    Toggle modal
+  </button>
+  {state.modalOpened && <ExperimentalModal
+    title='Ada Lovelace'
+    description={content.ada.short}
+    mobileFullscreen
```

Comment:
This prop is not very understandable IMHO, if we have a new modal, it could be time to find an alternative that is more understandable like isFullscreenOnMobile ?

Reply:
This prop is not used in the implementation. I think that in general, modals should be always fullscreen on mobile.

Reply:
We tend to do that (ie always be fullscreen on mobile) but I'm waiting for a full "GO" of @joel-costa on this point. But I agree with you ;)

Reply:
In fact it's a copy paste error on my own for the exemple. We already force mobileFullScreen on this component